ibmcloud/terraform/cluster/ansible/kata-playbook.yml

```diff
@@ -27,10 +18,6 @@
           - libdevmapper-dev
           - libseccomp-dev
           - pkg-config
-          - kubelet
-          - kubeadm
-          - kubectl
-          - kubernetes-cni
           - cri-tools
```

Comment:
The cri-tools package is only in the https://apt.kubernetes.io kubernetes-xenial/main apt repository, so if the kata enablement playbook needs to install the cri tools it needs to keep the 'Add Kubernetes apt repository signing key' and 'Add Kubernetes apt repository' tasks.

If cri-tools is used to debug the kubelet, does installing the cri-tools package only belong in he Kubernetes enablement playbook?

Reply:
Tested and successfully reran Terraform template.
Removed the `cri-tools` dependency from the kata-playbook.
Test 1 result:
   Apply complete!